### ISSUE TITLE
[libretro] [linux] Triggers not working

### ISSUE THREAD

**Issue by BPaden** [inolen/redream#102]
Triggers aren't working for me when using libretro. If I build redream standalone, the triggers work, and other libretro cores also work. Heres a log:

```
RetroArch [INFO] :: Redirecting save file to "/data/roms/dreamcast/stc-ct2u.srm".
RetroArch [INFO] :: === Build =======================================
Capabilities: MMX MMXEXT SSE1 SSE2 SSE3 SSSE3 SSE4 
Built: Jun 22 2017
RetroArch [INFO] :: Version: 1.6.1
RetroArch [INFO] :: =================================================
RetroArch [INFO] :: [Config]: Loading default config.
RetroArch [INFO] :: [Config]: loading config from: (null).
RetroArch [INFO] :: Looking for config in: "/home/aaron/.config/retroarch/retroarch.cfg".
RetroArch [INFO] :: Loading dynamic libretro core from: "/home/aaron/.config/retroarch/cores/redream_libretro.so"
RetroArch [INFO] :: [overrides] no core-specific overrides found at /home/aaron/.config/retroarch/config/redream/redream.cfg.
RetroArch [INFO] :: [overrides] no game-specific overrides found at /home/aaron/.config/retroarch/config/redream/stc-ct2u.cfg.
RetroArch [INFO] :: Shaders: preset directory: /home/aaron/.config/retroarch/shaders/presets
RetroArch [INFO] :: Shaders: no game-specific preset found at /home/aaron/.config/retroarch/shaders/presets/redream/stc-ct2u.cgp.
RetroArch [INFO] :: Shaders: no game-specific preset found at /home/aaron/.config/retroarch/shaders/presets/redream/stc-ct2u.glslp.
RetroArch [INFO] :: Shaders: no core-specific preset found at /home/aaron/.config/retroarch/shaders/presets/redream/redream.cgp.
RetroArch [INFO] :: Shaders: no core-specific preset found at /home/aaron/.config/retroarch/shaders/presets/redream/redream.glslp.
RetroArch [INFO] :: Environ SET_VARIABLES.
RetroArch [INFO] :: Remaps: remap directory: /home/aaron/.config/retroarch/config/remaps
RetroArch [INFO] :: Remaps: no game-specific remap found at /home/aaron/.config/retroarch/config/remaps/redream/stc-ct2u.rmp.
RetroArch [INFO] :: Remaps: no core-specific remap found at /home/aaron/.config/retroarch/config/remaps/redream/redream.rmp.
RetroArch [INFO] :: Redirecting save file to "/data/roms/dreamcast/stc-ct2u.srm".
RetroArch [INFO] :: Environ SYSTEM_DIRECTORY: "/data/roms/bios".
RetroArch [INFO] :: Environ GET_LOG_INTERFACE.
RetroArch [INFO] :: Content loading skipped. Implementation will load it on its own.
RetroArch [INFO] :: Environ GET_VARIABLE redream_resolution:
RetroArch [INFO] :: 	640x480
RetroArch [INFO] :: Environ SET_INPUT_DESCRIPTORS:
RetroArch [INFO] :: 	RetroPad, User 1, Button "B (bottom)" => "A"
RetroArch [INFO] :: 	RetroPad, User 1, Button "Y (left)" => "X"
RetroArch [INFO] :: 	RetroPad, User 1, Button "Start" => "Start"
RetroArch [INFO] :: 	RetroPad, User 1, Button "D-Pad Up" => "D-Pad Up"
RetroArch [INFO] :: 	RetroPad, User 1, Button "D-Pad Down" => "D-Pad Down"
RetroArch [INFO] :: 	RetroPad, User 1, Button "D-Pad Left" => "D-Pad Left"
RetroArch [INFO] :: 	RetroPad, User 1, Button "D-Pad Right" => "D-Pad Right"
RetroArch [INFO] :: 	RetroPad, User 1, Button "A (right)" => "B"
RetroArch [INFO] :: 	RetroPad, User 1, Button "X (up)" => "Y"
RetroArch [INFO] :: 	RetroPad, User 1, Button "L2" => "L"
RetroArch [INFO] :: 	RetroPad, User 1, Button "R2" => "R"
RetroArch [INFO] :: 	RetroPad, User 2, Button "B (bottom)" => "A"
RetroArch [INFO] :: 	RetroPad, User 2, Button "Y (left)" => "X"
RetroArch [INFO] :: 	RetroPad, User 2, Button "Start" => "Start"
RetroArch [INFO] :: 	RetroPad, User 2, Button "D-Pad Up" => "D-Pad Up"
RetroArch [INFO] :: 	RetroPad, User 2, Button "D-Pad Down" => "D-Pad Down"
RetroArch [INFO] :: 	RetroPad, User 2, Button "D-Pad Left" => "D-Pad Left"
RetroArch [INFO] :: 	RetroPad, User 2, Button "D-Pad Right" => "D-Pad Right"
RetroArch [INFO] :: 	RetroPad, User 2, Button "A (right)" => "B"
RetroArch [INFO] :: 	RetroPad, User 2, Button "X (up)" => "Y"
RetroArch [INFO] :: 	RetroPad, User 2, Button "L2" => "L"
RetroArch [INFO] :: 	RetroPad, User 2, Button "R2" => "R"
RetroArch [INFO] :: 	RetroPad, User 3, Button "B (bottom)" => "A"
RetroArch [INFO] :: 	RetroPad, User 3, Button "Y (left)" => "X"
RetroArch [INFO] :: 	RetroPad, User 3, Button "Start" => "Start"
RetroArch [INFO] :: 	RetroPad, User 3, Button "D-Pad Up" => "D-Pad Up"
RetroArch [INFO] :: 	RetroPad, User 3, Button "D-Pad Down" => "D-Pad Down"
RetroArch [INFO] :: 	RetroPad, User 3, Button "D-Pad Left" => "D-Pad Left"
RetroArch [INFO] :: 	RetroPad, User 3, Button "D-Pad Right" => "D-Pad Right"
RetroArch [INFO] :: 	RetroPad, User 3, Button "A (right)" => "B"
RetroArch [INFO] :: 	RetroPad, User 3, Button "X (up)" => "Y"
RetroArch [INFO] :: 	RetroPad, User 3, Button "L2" => "L"
RetroArch [INFO] :: 	RetroPad, User 3, Button "R2" => "R"
RetroArch [INFO] :: 	RetroPad, User 4, Button "B (bottom)" => "A"
RetroArch [INFO] :: 	RetroPad, User 4, Button "Y (left)" => "X"
RetroArch [INFO] :: 	RetroPad, User 4, Button "Start" => "Start"
RetroArch [INFO] :: 	RetroPad, User 4, Button "D-Pad Up" => "D-Pad Up"
RetroArch [INFO] :: 	RetroPad, User 4, Button "D-Pad Down" => "D-Pad Down"
RetroArch [INFO] :: 	RetroPad, User 4, Button "D-Pad Left" => "D-Pad Left"
RetroArch [INFO] :: 	RetroPad, User 4, Button "D-Pad Right" => "D-Pad Right"
RetroArch [INFO] :: 	RetroPad, User 4, Button "A (right)" => "B"
RetroArch [INFO] :: 	RetroPad, User 4, Button "X (up)" => "Y"
RetroArch [INFO] :: 	RetroPad, User 4, Button "L2" => "L"
RetroArch [INFO] :: 	RetroPad, User 4, Button "R2" => "R"
RetroArch [INFO] :: Environ SET_HW_RENDER.
RetroArch [INFO] :: Requesting core OpenGL context (3.3).
RetroArch [libretro INFO] :: boot_load_rom loaded '/data/roms/bios/dc/dc_boot.bin'
RetroArch [libretro INFO] :: pvr_reconfigure_spg mode=VGA pixel_clock=13500000 line_clock=15734 vcount=262 hcount=857 interlace=0 vbstart=260 vbend=336
RetroArch [libretro INFO] :: loading /data/roms/dreamcast/stc-ct2u.cdi
RetroArch [libretro INFO] :: cdi_parse version 3 detected
RetroArch [libretro INFO] :: cdi_parse found 2 sessions
RetroArch [libretro INFO] :: cdi_parse_track track=1 fad=150 mode=CDDA/2352
RetroArch [libretro INFO] :: cdi_parse_track track=2 fad=11852 mode=M2F1/2336
RetroArch [libretro INFO] :: gdrom_set_disc CRAZY TAXI 2 V1.004 - MK-51136
RetroArch [INFO] :: Skipping SRAM load..
RetroArch [INFO] :: CHEEVOS system RAM: (nil) 0
RetroArch [INFO] :: Version of libretro API: 1
RetroArch [INFO] :: Compiled against API: 1
RetroArch [INFO] :: CHEEVOS save RAM:   (nil) 0
RetroArch [INFO] :: CHEEVOS video RAM:  (nil) 0
RetroArch [INFO] :: CHEEVOS RTC:        (nil) 0
RetroArch [INFO] :: [Audio]: Set audio input rate to: 44055.95 Hz.
RetroArch [INFO] :: Found shader "/home/aaron/.config/retroarch/shaders/retroarch.glslp"
RetroArch [INFO] :: [Video]: Video @ fullscreen
RetroArch [INFO] :: [Video]: Using HW render, OpenGL driver forced.
RetroArch [ERROR] :: [Wayland]: Failed to connect to Wayland server.
RetroArch [INFO] :: [GLX]: GLX_OML_sync_control and GLX_MESA_swap_control supported, using better swap control method...
RetroArch [INFO] :: [GL]: Found GL context: x
RetroArch [INFO] :: [GL]: Detecting screen resolution 1440x900.
RetroArch [INFO] :: [XINERAMA]: Xinerama version: 1.1.
RetroArch [INFO] :: [XINERAMA]: Xinerama screens: 1.
RetroArch [INFO] :: [GLX]: Using Xinerama on screen #0.
RetroArch [INFO] :: [GLX]: X = 0, Y = 0, W = 1440, H = 900.
RetroArch [INFO] :: [GLX]: Using windowed fullscreen.
RetroArch [INFO] :: [GLX]: Found swap function: glXSwapIntervalEXT.
RetroArch [INFO] :: [GLX]: glXSwapIntervalEXT(1)
RetroArch [INFO] :: [GL]: Vendor: X.Org, Renderer: Gallium 0.4 on AMD OLAND (DRM 2.49.0 / 4.11.7-1-ARCH, LLVM 4.0.1).
RetroArch [INFO] :: [GL]: Version: 4.5 (Core Profile) Mesa 17.1.4.
RetroArch [INFO] :: [GL]: Using Core GL context.
RetroArch [INFO] :: [GL]: Using resolution 1440x900
RetroArch [INFO] :: [GL]: Default shader backend found: glsl.
RetroArch [INFO] :: [Shader driver]: Using GLSL shader backend.
RetroArch [INFO] :: [GLSL]: Checking GLSL shader support ...
RetroArch [INFO] :: Found #pragma parameter Screen curvature - horizontal (CURVATURE_X) 0.100000 0.000000 1.000000 0.010000
RetroArch [INFO] :: Found #pragma parameter Screen curvature - vertical (CURVATURE_Y) 0.150000 0.000000 1.000000 0.010000
RetroArch [INFO] :: Found #pragma parameter Mask brightness (MASK_BRIGHTNESS) 0.700000 0.000000 1.000000 0.010000
RetroArch [INFO] :: Found #pragma parameter Scanline weight (SCANLINE_WEIGHT) 6.000000 0.000000 15.000000 0.100000
RetroArch [INFO] :: Found #pragma parameter Scanline gap brightness (SCANLINE_GAP_BRIGHTNESS) 0.120000 0.000000 1.000000 0.010000
RetroArch [INFO] :: Found #pragma parameter Bloom factor (BLOOM_FACTOR) 1.500000 0.000000 5.000000 0.010000
RetroArch [INFO] :: Found #pragma parameter Input gamma (INPUT_GAMMA) 2.400000 0.000000 5.000000 0.010000
RetroArch [INFO] :: Found #pragma parameter Output gamma (OUTPUT_GAMMA) 2.200000 0.000000 5.000000 0.010000
RetroArch [INFO] :: [GLSL]: Found GLSL vertex shader.
RetroArch [INFO] :: [GLSL]: Using GLSL version 330.
RetroArch [INFO] :: [GLSL]: Found GLSL fragment shader.
RetroArch [INFO] :: [GLSL]: Using GLSL version 330.
RetroArch [INFO] :: [GLSL]: Linking GLSL program.
RetroArch [INFO] :: [GLSL]: Found GLSL vertex shader.
RetroArch [INFO] :: [GLSL]: Using GLSL version 130.
RetroArch [INFO] :: [GLSL]: Found GLSL fragment shader.
RetroArch [INFO] :: [GLSL]: Using GLSL version 130.
RetroArch [INFO] :: [GLSL]: Linking GLSL program.
RetroArch [INFO] :: [GLSL]: Found GLSL vertex shader.
RetroArch [INFO] :: [GLSL]: Using GLSL version 130.
RetroArch [INFO] :: [GLSL]: Found GLSL fragment shader.
RetroArch [INFO] :: [GLSL]: Using GLSL version 130.
RetroArch [INFO] :: [GLSL]: Linking GLSL program.
RetroArch [INFO] :: [GLSL]: Found GLSL vertex shader.
RetroArch [INFO] :: [GLSL]: Using GLSL version 330.
RetroArch [INFO] :: [GLSL]: Found GLSL fragment shader.
RetroArch [INFO] :: [GLSL]: Using GLSL version 330.
RetroArch [INFO] :: [GLSL]: Linking GLSL program.
RetroArch [INFO] :: [GLSL]: Found GLSL vertex shader.
RetroArch [INFO] :: [GLSL]: Using GLSL version 330.
RetroArch [INFO] :: [GLSL]: Found GLSL fragment shader.
RetroArch [INFO] :: [GLSL]: Using GLSL version 330.
RetroArch [INFO] :: [GLSL]: Linking GLSL program.
RetroArch [INFO] :: [GLSL]: Found GLSL vertex shader.
RetroArch [INFO] :: [GLSL]: Using GLSL version 330.
RetroArch [INFO] :: [GLSL]: Found GLSL fragment shader.
RetroArch [INFO] :: [GLSL]: Using GLSL version 330.
RetroArch [INFO] :: [GLSL]: Linking GLSL program.
RetroArch [INFO] :: [GLSL]: Found GLSL vertex shader.
RetroArch [INFO] :: [GLSL]: Using GLSL version 330.
RetroArch [INFO] :: [GLSL]: Found GLSL fragment shader.
RetroArch [INFO] :: [GLSL]: Using GLSL version 330.
RetroArch [INFO] :: [GLSL]: Linking GLSL program.
RetroArch [INFO] :: [GLSL]: Found GLSL vertex shader.
RetroArch [INFO] :: [GLSL]: Using GLSL version 330.
RetroArch [INFO] :: [GLSL]: Found GLSL fragment shader.
RetroArch [INFO] :: [GLSL]: Using GLSL version 330.
RetroArch [INFO] :: [GLSL]: Linking GLSL program.
RetroArch [INFO] :: [GLSL]: Found GLSL vertex shader.
RetroArch [INFO] :: [GLSL]: Using GLSL version 330.
RetroArch [INFO] :: [GLSL]: Found GLSL fragment shader.
RetroArch [INFO] :: [GLSL]: Using GLSL version 330.
RetroArch [INFO] :: [GLSL]: Linking GLSL program.
RetroArch [INFO] :: [GLSL]: Found GLSL vertex shader.
RetroArch [INFO] :: [GLSL]: Using GLSL version 330.
RetroArch [INFO] :: [GLSL]: Found GLSL fragment shader.
RetroArch [INFO] :: [GLSL]: Using GLSL version 330.
RetroArch [INFO] :: [GLSL]: Linking GLSL program.
RetroArch [INFO] :: [GL]: Using 1 textures.
RetroArch [INFO] :: [GL]: Loaded 3 program(s).
RetroArch [INFO] :: [GL]: Using GL_RGB565 for texture uploads.
RetroArch [INFO] :: [GL]: Creating FBO 0 @ 1024x1024
RetroArch [INFO] :: [GL]: Creating FBO 1 @ 512x1024
RetroArch [INFO] :: [GL]: FBO pass #0 is floating-point.
RetroArch [INFO] :: [GL]: Initializing HW render (1024 x 1024).
RetroArch [INFO] :: [GL]: Max texture size: 16384 px, renderbuffer size: 16384 px.
RetroArch [INFO] :: [GL]: Supports FBO (render-to-texture).
RetroArch [WARN] :: [udev]: Couldn't open any keyboard, mouse or touchpad. Are permissions set correctly for /dev/input/event*?
RetroArch [INFO] :: [udev]: Plugged pad: Microsoft X-Box 360 pad (1118:654) on port #0.
RetroArch [INFO] :: [udev]: Pad #0 (/dev/input/event14) supports force feedback.
RetroArch [INFO] :: [udev]: Pad #0 (/dev/input/event14) supports 16 force feedback effects.
RetroArch [INFO] :: [Autodetect]: no profiles found for Microsoft X-Box 360 pad (1118/654).
RetroArch [INFO] :: [Joypad]: Found joypad driver: "udev".
RetroArch [INFO] :: [Font]: Using font rendering backend: freetype.
RetroArch [INFO] :: [DBus]: Suspended screensaver via DBus.
RetroArch [INFO] :: [PulseAudio]: Requested 24576 bytes buffer, got 18432.
RetroArch [INFO] :: [Menu]: Found menu display driver: "menu_display_gl".
RetroArch [INFO] :: Found #pragma parameter Screen curvature - horizontal (CURVATURE_X) 0.100000 0.000000 1.000000 0.010000
RetroArch [INFO] :: Found #pragma parameter Screen curvature - vertical (CURVATURE_Y) 0.150000 0.000000 1.000000 0.010000
RetroArch [INFO] :: Found #pragma parameter Mask brightness (MASK_BRIGHTNESS) 0.700000 0.000000 1.000000 0.010000
RetroArch [INFO] :: Found #pragma parameter Scanline weight (SCANLINE_WEIGHT) 6.000000 0.000000 15.000000 0.100000
RetroArch [INFO] :: Found #pragma parameter Scanline gap brightness (SCANLINE_GAP_BRIGHTNESS) 0.120000 0.000000 1.000000 0.010000
RetroArch [INFO] :: Found #pragma parameter Bloom factor (BLOOM_FACTOR) 1.500000 0.000000 5.000000 0.010000
RetroArch [INFO] :: Found #pragma parameter Input gamma (INPUT_GAMMA) 2.400000 0.000000 5.000000 0.010000
RetroArch [INFO] :: Found #pragma parameter Output gamma (OUTPUT_GAMMA) 2.200000 0.000000 5.000000 0.010000
RetroArch [INFO] :: [Font]: Using font rendering backend: freetype.
RetroArch [INFO] :: [Font]: Using font rendering backend: freetype.
RetroArch [WARN] :: Input device ID 5 is unknown to this libretro implementation. Using RETRO_DEVICE_JOYPAD.
RetroArch [INFO] :: Found #pragma parameter Screen curvature - horizontal (CURVATURE_X) 0.100000 0.000000 1.000000 0.010000
RetroArch [INFO] :: Found #pragma parameter Screen curvature - vertical (CURVATURE_Y) 0.150000 0.000000 1.000000 0.010000
RetroArch [INFO] :: Found #pragma parameter Mask brightness (MASK_BRIGHTNESS) 0.700000 0.000000 1.000000 0.010000
RetroArch [INFO] :: Found #pragma parameter Scanline weight (SCANLINE_WEIGHT) 6.000000 0.000000 15.000000 0.100000
RetroArch [INFO] :: Found #pragma parameter Scanline gap brightness (SCANLINE_GAP_BRIGHTNESS) 0.120000 0.000000 1.000000 0.010000
RetroArch [INFO] :: Found #pragma parameter Bloom factor (BLOOM_FACTOR) 1.500000 0.000000 5.000000 0.010000
RetroArch [INFO] :: Found #pragma parameter Input gamma (INPUT_GAMMA) 2.400000 0.000000 5.000000 0.010000
RetroArch [INFO] :: Found #pragma parameter Output gamma (OUTPUT_GAMMA) 2.200000 0.000000 5.000000 0.010000
RetroArch [INFO] :: Loading history file: [/home/aaron/.config/retroarch/content_history.lpl].
RetroArch [INFO] :: Loading history file: [/home/aaron/.config/retroarch/content_music_history.lpl].
RetroArch [INFO] :: Loading history file: [/home/aaron/.config/retroarch/content_video_history.lpl].
RetroArch [INFO] :: Loading history file: [/home/aaron/.config/retroarch/content_image_history.lpl].
RetroArch [INFO] :: [GL]: VSync => on
RetroArch [INFO] :: [GLX]: glXSwapIntervalEXT(1)
RetroArch [INFO] :: [PulseAudio]: Unpausing.
RetroArch [libretro INFO] :: sh4_ccn_reset
RetroArch [libretro INFO] :: pvr_reconfigure_spg mode=VGA pixel_clock=13500000 line_clock=15734 vcount=262 hcount=857 interlace=0 vbstart=260 vbend=336
RetroArch [libretro INFO] :: pvr_reconfigure_spg mode=NTSC pixel_clock=13500000 line_clock=31468 vcount=524 hcount=857 interlace=1 vbstart=260 vbend=336
RetroArch [libretro INFO] :: arm7_reset
RetroArch [libretro INFO] :: sh4_ccn_reset
RetroArch [libretro INFO] :: sh4_ccn_reset
RetroArch [libretro INFO] :: sh4_ccn_reset
RetroArch [libretro INFO] :: pvr_reconfigure_spg mode=NTSC pixel_clock=13500000 line_clock=31468 vcount=524 hcount=857 interlace=1 vbstart=516 vbend=36
RetroArch [libretro INFO] :: pvr_reconfigure_spg mode=NTSC pixel_clock=13500000 line_clock=31468 vcount=524 hcount=857 interlace=1 vbstart=516 vbend=36
RetroArch [libretro INFO] :: pvr_reconfigure_spg mode=NTSC pixel_clock=13500000 line_clock=31468 vcount=524 hcount=857 interlace=1 vbstart=516 vbend=36
RetroArch [libretro INFO] :: pvr_reconfigure_spg mode=NTSC pixel_clock=13500000 line_clock=31468 vcount=524 hcount=857 interlace=1 vbstart=516 vbend=36
RetroArch [libretro INFO] :: pvr_reconfigure_spg mode=NTSC pixel_clock=13500000 line_clock=31468 vcount=524 hcount=857 interlace=1 vbstart=516 vbend=36
RetroArch [libretro INFO] :: arm7_reset
RetroArch [libretro INFO] :: pvr_reconfigure_spg mode=NTSC pixel_clock=13500000 line_clock=31468 vcount=524 hcount=857 interlace=1 vbstart=516 vbend=36
RetroArch [libretro INFO] :: sh4_ccn_reset
RetroArch [libretro INFO] :: sh4_ccn_reset
RetroArch [libretro INFO] :: sh4_ccn_reset
RetroArch [libretro INFO] :: pvr_reconfigure_spg mode=NTSC pixel_clock=13500000 line_clock=31468 vcount=524 hcount=857 interlace=1 vbstart=516 vbend=36
RetroArch [libretro INFO] :: pvr_reconfigure_spg mode=NTSC pixel_clock=13500000 line_clock=31468 vcount=524 hcount=857 interlace=1 vbstart=516 vbend=36
RetroArch [libretro INFO] :: pvr_reconfigure_spg mode=NTSC pixel_clock=13500000 line_clock=31468 vcount=524 hcount=857 interlace=1 vbstart=516 vbend=36
RetroArch [libretro INFO] :: sh4_ccn_reset
RetroArch [libretro INFO] :: sh4_ccn_reset
RetroArch [libretro INFO] :: sh4_ccn_reset
RetroArch [libretro INFO] :: sh4_ccn_reset
RetroArch [libretro INFO] :: pvr_reconfigure_spg mode=NTSC pixel_clock=13500000 line_clock=31468 vcount=524 hcount=857 interlace=1 vbstart=516 vbend=36
RetroArch [libretro INFO] :: pvr_reconfigure_spg mode=NTSC pixel_clock=13500000 line_clock=31468 vcount=524 hcount=857 interlace=1 vbstart=516 vbend=36
RetroArch [libretro INFO] :: pvr_reconfigure_spg mode=NTSC pixel_clock=13500000 line_clock=31468 vcount=524 hcount=857 interlace=1 vbstart=516 vbend=36
RetroArch [libretro INFO] :: pvr_reconfigure_spg mode=NTSC pixel_clock=13500000 line_clock=31468 vcount=524 hcount=857 interlace=1 vbstart=516 vbend=36
RetroArch [libretro INFO] :: pvr_reconfigure_spg mode=NTSC pixel_clock=13500000 line_clock=31468 vcount=524 hcount=857 interlace=1 vbstart=516 vbend=36
RetroArch [libretro INFO] :: arm7_reset
RetroArch [INFO] :: [GL]: VSync => on
RetroArch [INFO] :: [GLX]: glXSwapIntervalEXT(1)
RetroArch [INFO] :: [PulseAudio]: Pausing.
RetroArch [INFO] :: [PulseAudio]: Unpausing.
RetroArch [INFO] :: [PulseAudio]: Unpausing.
RetroArch [INFO] :: [PulseAudio]: Unpausing.
RetroArch [INFO] :: [Config]: Saved new config to "/home/aaron/.config/retroarch/retroarch.cfg".
RetroArch [INFO] :: [PulseAudio]: Pausing.
RetroArch [INFO] :: Saved core options file to "/home/aaron/.config/retroarch/retroarch-core-options.cfg"
RetroArch [INFO] :: [XINERAMA]: Xinerama version: 1.1.
RetroArch [INFO] :: [XINERAMA]: Xinerama screens: 1.
RetroArch [INFO] :: [XINERAMA]: Saved monitor #0.
RetroArch [INFO] :: [Video]: Does not have enough samples for monitor refresh rate estimation. Requires to run for at least 4096 frames.
RetroArch [INFO] :: [Audio]: Average audio buffer saturation: 67.85 %, standard deviation (percentage points): 11.95 %.
RetroArch [INFO] :: [Audio]: Amount of time spent close to underrun: 0.00 %. Close to blocking: 29.57 %.
RetroArch [INFO] :: [Video]: Does not have enough samples for monitor refresh rate estimation. Requires to run for at least 4096 frames.
RetroArch [INFO] :: [Audio]: Average audio buffer saturation: 67.85 %, standard deviation (percentage points): 11.95 %.
RetroArch [INFO] :: [Audio]: Amount of time spent close to underrun: 0.00 %. Close to blocking: 29.57 %.
```

**Comment by inolen** [comment]
Thanks, I'll be taking a look at some RA issues tonight, will look at this as well.

**Comment by inolen** [comment]
Should be resolved with ba07ae81267aa08243e4a0b3e1cd1c487bf7a521.
